src/run_sp_tests.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+# Run all the shortest path tests in the sp_tests directory. See README.md
+# for information on how to write tests.
+
+# Set some strings to help us format our results.
+green_format='\033[0;32m'
+red_format='\033[0;31m'
+reset_format='\033[0m'
+
+# Specify that all the shortest path tests are in the sp_tests directory.
+sp_tests_dir="sp_tests"
```

Comment:
Better to be explicit.
I think it's Ok to use the "sp" abbreviation local to this script because the narrower context makes the meaning clear, so
sp_tests_dir="shortest_path_tests"

Reply:
Let me know what you think about the updated code. I _think_ that I understood what you are asking and made the appropriate updates.